web/client/components/map/openlayers/__tests__/ScaleBar-test.jsx

```diff
@@ -5,26 +5,27 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
-var React = require('react');
-var ReactDOM = require('react-dom');
-var ol = require('openlayers');
-var ScaleBar = require('../ScaleBar');
-var expect = require('expect');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import expect from 'expect';
+import ScaleBar from '../ScaleBar';
+
+import { Map, View } from 'ol';
 
 describe('Openlayers ScaleBar component', () => {
     let map;
     beforeEach((done) => {
         document.body.innerHTML = '<div id="map"></div><div id="container"></div>';
-        map = new ol.Map({
+        map = new Map({
             layers: [
             ],
-            controls: ol.control.defaults({
-                attributionOptions: /** @type {olx.control.AttributionOptions} */ {
+            /*controls: ol.control.defaults({
```

Comment:
There is commented code, maybe we can remove it?

Reply:
@allyoucanmap I should have removed them now